src/vdaf/dummy.rs

```diff
@@ -331,3 +333,86 @@ impl Encode for AggregateShare {
         self.0.encoded_len()
     }
 }
+
+/// Returns the aggregate result that the dummy VDAF would compute over the provided measurements,
+/// for the provided aggregation parameter.
+pub fn expected_aggregate_result<M>(aggregation_parameter: u8, measurements: M) -> u64
+where
+    M: IntoIterator<Item = u8>,
+{
+    (measurements.into_iter().map(u64::from).sum::<u64>()) % u64::from(u8::MAX)
+        + u64::from(aggregation_parameter)
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::vdaf::{test_utils::run_vdaf_sharded, Client};
+    use rand::prelude::*;
+
+    fn run_test(rounds: u32, aggregation_parameter: u8) {
+        let vdaf = Vdaf::new(rounds);
+        let mut verify_key = [0; 0];
+        thread_rng().fill(&mut verify_key[..]);
+        let measurements = [1, 2, 3, 4, 5];
```

Comment:
The tests pass even without the above change since the measurements aren't large enough exercise the modulo. We could get clever with randomness and multiple rounds of testing, but I think it's sufficient to just make the measurements larger.
```suggestion
        let measurements = [10, 20, 30, 40, 50, 60, 70, 80, 90, 100];
```

Reply:
Doing this also revealed a bug where I used the wrong modulus in `expected_aggregate_result`. Thanks!